docs/presetGuides.md

```diff
@@ -297,3 +253,129 @@ nodes:
+```
+
+Remember to always keep your custom preset, the target's `addresses.yml`, and `preset.yml` privately!
+
+### Specify the Nodes' Public Keys.
+
+In addition to the above, you can provide the accounts Public Keys instead of Private keys in your custom preset.
+This adds a level of security as the private keys aren't in the custom preset. 
+Bootstrap will prompt for them when performing the different operations requiring user intervention when running the commands.
+
+Example:
+
+#### **`plain-custom-preset.yml`**
+```yaml
+nodes:
+    - voting: true
+      mainPublicKey: 93F9D99300DD04B66E727CA72667E5A2F6549A28AB8676B69ECAF30455C332E6
+      transportPublicKey: C9767496987222790518114049299DD52114BF2A8F7E5F4B70BB2B6365FAFD34
+      remotePublicKey: 2350F44F043DB9290390C59F1E1C2F06FCC1675E8B5F4CBDC00DD9DF4428E8FE
+      vrfPublicKey: 7834C8AAEFB09402DA32B22768D7CF9DCAC78A7774392E4452F07DDF776E088D
+      votingPublicKey: 4558724BE79FD0FD08A279878AF92F999B28D5622D3531CE9EFBC06466CDDF9C
+```
+
+In this case, this custom preset does not need to be encrypted as it doesn't contain any private information. Bootstrap will prompt for all private keys.
+
+```
+symbol-bootstrap start --preset testnet --assembly dual --customPreset plain-custom-preset.yml
+```
+
+**Hint:**
+
+You can provide some Private Keys, and some Public Keys in your custom preset. Remember to encrypt when Private Keys are provided!
+
+### Private Key Security Mode.
+
+By default, Bootstrap encrypts and stores Private Keys in the `./target/addresses.yml` file. This allows Bootstrap to perform the different operations by just asking for a password.
+
+It may be required that the Node cannot store the Main or any Private Key, even in encrypted form. 
+
+If this is you case, you can tell Bootstrap to not store these keys with the `privateKeySecurityMode` preset property. 
+
+The `privateKeySecurityMode` defines which Private Keys can be encrypted and stored in the `target/addresses.yml`:
+-   `ENCRYPT`: All private Keys are encrypted and stored in the target's `addresses.yml` file. Bootstrap will have them to be used when required. This is Bootstrap's default behaviour.
+-   `PROMPT_MAIN`: Main Private Keys are not stored in the target's `addresses.yml` file. Bootstrap will prompt for the Main Private Key when generating certificates, or transactions need to be signed in the `link` and `enrolProgram` commands.
+-   `PROMPT_ALL`: No Private Key is stored in the target's `addresses.yml` file. Bootstrap will prompt for the Private Keys when they are required in the different commands.
+
+When using the `PROMPT_*` security mode, Bootstrap may ask for Private Keys when running the different commands. It may not be suitable for automatic scripting.  
+
+Bootstrap will reject `PROMPT_*` security mode when an account is randomly generated. Otherwise, the generated account won't be stored anywhere.
```

Comment:
```suggestion
When using the `PROMPT` security modes Bootstrap may ask for private keys when running the different commands. This may not be suitable for automatic scripting.

Bootstrap will reject `PROMPT` security modes when an account is being randomly generated. Otherwise, the generated account wouldn't be stored anywhere.
```